aistweet/tweeter.py

```diff
@@ -144,10 +142,11 @@ def snap_and_tweet(self, mmsi, depth):
             # tweet the image with info
             lat, lon = self.tracker.center_coords(mmsi)
             try:
-                self.twitter.update_status_with_media(
-                    self.generate_text(mmsi), image_path, lat=lat, long=lon
+                self.twitter.tweet(
+                    text=self.generate_text(mmsi),
+                    image_path=image_path, lat=lat, long=lon
                 )
-            except tweepy.errors.TweepyException as e:
+            except Exception as e:
```

Comment:
Does tweeter-basic throw a specific exception type?

Reply:
Not really, it just throws whatever it gets - most commonly an HTTP error: https://requests.readthedocs.io/en/latest/_modules/requests/exceptions/